docs/framework/wcf/how-to-use-a-wcf-client.md

```diff
@@ -1,143 +1,150 @@
+            double value1 = 100.00D;
+            double value2 = 15.99D;
+            double result = client.Add(value1, value2);
+            Console.WriteLine("Add({0},{1}) = {2}", value1, value2, result);
+
+            // Call the Subtract service operation.
+            value1 = 145.00D;
+            value2 = 76.54D;
+            result = client.Subtract(value1, value2);
+            Console.WriteLine("Subtract({0},{1}) = {2}", value1, value2, result);
+
+            // Call the Multiply service operation.
+            value1 = 9.00D;
+            value2 = 81.25D;
+            result = client.Multiply(value1, value2);
+            Console.WriteLine("Multiply({0},{1}) = {2}", value1, value2, result);
+
+            // Call the Divide service operation.
+            value1 = 22.00D;
+            value2 = 7.00D;
+            result = client.Divide(value1, value2);
+            Console.WriteLine("Divide({0},{1}) = {2}", value1, value2, result);
+
+            //Step 3: Closing the client gracefully closes the connection and cleans up resources.
+            client.Close();
+        }
+    }
+}
+```
+
+```vb
+Imports System
+Imports System.Collections.Generic
+Imports System.Text
+Imports System.ServiceModel
+Imports GettingStartedClientVB2.ServiceReference1
+
+Module Module1
+
+    Sub Main()
+        ' Step 1: Create an instance of the WCF proxy
+        Dim Client As New CalculatorClient()
+
+        'Step 2: Call the service operations.
+        'Call the Add service operation.
+        Dim value1 As Double = 100D
+        Dim value2 As Double = 15.99D
+        Dim result As Double = Client.Add(value1, value2)
+        Console.WriteLine("Add({0},{1}) = {2}", value1, value2, result)
+
+        'Call the Subtract service operation.
+        value1 = 145D
+        value2 = 76.54D
+        result = Client.Subtract(value1, value2)
+        Console.WriteLine("Subtract({0},{1}) = {2}", value1, value2, result)
+
+        'Call the Multiply service operation.
+        value1 = 9D
+        value2 = 81.25D
+        result = Client.Multiply(value1, value2)
+        Console.WriteLine("Multiply({0},{1}) = {2}", value1, value2, result)
+
+        'Call the Divide service operation.
+        value1 = 22D
+        value2 = 7D
+        result = Client.Divide(value1, value2)
+        Console.WriteLine("Divide({0},{1}) = {2}", value1, value2, result)
+
+        ' Step 3: Closing the client gracefully closes the connection and cleans up resources.
+        Client.Close()
+
+        Console.WriteLine()
+        Console.WriteLine("Press <ENTER> to terminate client.")
+        Console.ReadLine()
+
+    End Sub
+
+End Module
+```
+
+Notice the using or imports statement that imports the GettingStartedClient.ServiceReference1. This imports the code generated by Add Service Reference in Visual Studio. The code instantiates the WCF proxy and then calls each of the service operations exposed by the calculator service, closes the proxy and terminates.
```

Comment:
using, imports, and GettingStartedClientServiceReference1 can all be fenced.
Add Service Reference can be bolded.
comma after "closed the proxy"